textures/metro_control_panel_001_pegboard_wip.py

```diff
@@ -0,0 +1,136 @@
+"""A collection of stuff related to https://stalburg.net/Body_message#Pegs."""
+from infra.encodings.binary import binary_decode
+from textures.metro_control_panel_001_pegboard import (metro_control_panel_001_pegboard, _invert_bool_list,
+                                                       _bools_to_binary_str, _bools_to_nor_mask, _pegs_to_bool_list)
+
+mp = metro_control_panel_001_pegboard
+
+
+if __name__ == "__main__":
+    def transform_pegs(a: str, mask: str, b: str) -> str:
+        a_ = _pegs_to_bool_list(a)
+        mask_ = _bools_to_nor_mask(mask)
+        b_ = _pegs_to_bool_list(b)
+        transformed = [op(first, second) for first, op, second in zip(a_, mask_, b_)]
+        bytes_str = binary_decode(_bools_to_binary_str(transformed))
```

Comment:
Converting a sequence of booleans to a string and then back to a sequence of bytes seems a bit complicated.